torch/sparse/semi_structured.py

```diff
@@ -304,6 +337,55 @@ def __torch_dispatch__(cls, func, types, args, kwargs) -> Any:
                 transposed=not args[0].transposed,
             )
 
+        if func is torch.ops.prims.convert_element_type.default:
```

Comment:
@bdhirsh These are the ops I added to get this test to "work", you can comment these out to see where it fails earlier in the chain.